Source/SwiftLintFramework/Rules/RuleConfigurations/PrivateUnitTestConfiguration.swift

```diff
@@ -36,13 +36,12 @@ public struct PrivateUnitTestConfiguration: RuleConfiguration, Equatable {
     }
 
     public mutating func applyConfiguration(configuration: AnyObject) throws {
-        guard let configurationDict = configuration as? [String: AnyObject],
-            regexString = configurationDict["regex"] as? String else {
-                throw ConfigurationError.UnknownConfiguration
+        guard let configurationDict = configuration as? [String: AnyObject] else {
```

Comment:
Never mind, I just saw your side note in #786. Could you please add a bug fix entry in the changelog to reflect this fix? 😄